### PR TITLE
[Feature] Add FastFCN

### PR DESCRIPTION
**Pull request by justld**
添加模型 Fastfcn: Rethinking dilated convolution in the backbone for semantic segmentation

### CODE REVIEW

File: configs/fastfcn/README.md
Comment (shiyutang) [inline configs/fastfcn/README.md]: The reference is not complete, please change to : Wu, Huikai, Junge Zhang, Kaiqi Huang, Kongming Liang, and Yizhou Yu. "Fastfcn: Rethinking dilated convolution in the backbone for semantic segmentation." arXiv preprint arXiv:1903.11816 (2019).

File: configs/fastfcn/fastfcn_resnet50_os8_ade20k_480x480_120k.yml
Comment (shiyutang) [inline configs/fastfcn/fastfcn_resnet50_os8_ade20k_480x480_120k.yml]: remove annotation plz.

File: paddleseg/models/fastfcn.py
Comment (shiyutang) [inline paddleseg/models/fastfcn.py]: nn.Sequential seems to be unnecessary.

**Review by shiyutang — CHANGES_REQUESTED** [review]

File: configs/fastfcn/README.md
Comment (shiyutang) [inline configs/fastfcn/README.md]: plz add the mIOU results, including flip and ms+flip

File: configs/fastfcn/fastfcn_resnet50_os8_ade20k_480x480_120k.yml
Comment (justld) [reply configs/fastfcn/fastfcn_resnet50_os8_ade20k_480x480_120k.yml]: removed.

File: configs/fastfcn/README.md
Comment (justld) [reply configs/fastfcn/README.md]: changed.

File: paddleseg/models/fastfcn.py
Comment (justld) [reply paddleseg/models/fastfcn.py]: removed

File: configs/fastfcn/README.md
Comment (justld) [reply configs/fastfcn/README.md]: added.

**Comment by shiyutang** [comment]
The test result of retrained model is live up to expectation.
image

**Review by shiyutang — DISMISSED** [review]
LGTM

**Review by shiyutang — APPROVED** [review]
LGTM after merge

**Pull request merged**
